docs/src/introduction/quickstart.md

```diff
@@ -376,83 +229,122 @@ specify the initial data of the instance as payload when we start the instance.
+    "activityInstanceKey": 21
+  },
+  "customHeaders": "{\"method\":\"VISA\"}",
+  "worker": "zbctl",
+  "retries": 3,
+  "deadline": 1539604800511,
+  "payload": "{\"orderId\":1234}"
+}
+```
+And complete it by its job key `12`.
 ```
-./bin/zbctl --topic quickstart subscribe job --jobType payment-service cat &
-./bin/zbctl --topic quickstart subscribe job --jobType inventory-service cat &
-./bin/zbctl --topic quickstart subscribe job --jobType shipment-service cat &
+/bin/zbctl complete job 12
+```
+```
+2018/10/15 13:55:08 Completed job with key 12 and payload {}
 ```
 
-To verify that our workflow instance was completed after all jobs were
-processed we can again open a topic subscription. The last event should
-indicate that the workflow instance was completed.
+Next activate a job for the `inventory-service` type.
 
 ```
-./bin/zbctl --topic quickstart subscribe topic
+./bin/zbctl activate jobs inventory-service
 ```
 ```
-{"Intent":"END_EVENT_OCCURRED","Key":4294983224,"PartitionID":2,"Position":4294983224,"Type":"WORKFLOW_INSTANCE"}
+2018/10/15 13:55:18 Activated 1 for type inventory-service
+2018/10/15 13:55:18 Job 1 / 1
 {
-  "ActivityID": "",
-  "BPMNProcessID": "order-process",
-  "Payload": "gadvcmRlcklky0CTSAAAAAAA",
-  "Version": 1,
-  "WorkflowInstanceKey": 4294971952,
-  "WorkflowKey": 1
+  "key": 22,
+  "type": "inventory-service",
+  "jobHeaders": {
+    "workflowInstanceKey": 6,
+    "bpmnProcessId": "order-process",
+    "workflowDefinitionVersion": 1,
+    "workflowKey": 1,
+    "activityId": "fetch-items",
+    "activityInstanceKey": 31
+  },
+  "customHeaders": "{}",
+  "worker": "zbctl",
+  "retries": 3,
+  "deadline": 1539604818042,
+  "payload": "{\"orderId\":1234}"
 }
+```
+
+And complete it by its job key `22`.
+```
+./bin/zbctl complete job 22
+```
+```
+2018/10/15 13:55:20 Completed job with key 22 and payload {}
+```
 
-{"Intent":"COMPLETED","Key":4294971952,"PartitionID":2,"Position":4294983464,"Type":"WORKFLOW_INSTANCE"}
+Last activate a job for the `shipment-service` type.
+```
+./bin/zbctl activate jobs shipment-service
+```
+```
+2018/10/15 13:55:31 Activated 1 for type shipment-service
+2018/10/15 13:55:31 Job 1 / 1
 {
-  "ActivityID": "",
-  "BPMNProcessID": "order-process",
-  "Payload": "gadvcmRlcklky0CTSAAAAAAA",
-  "Version": 1,
-  "WorkflowInstanceKey": 4294971952,
-  "WorkflowKey": 1
+  "key": 32,
+  "type": "shipment-service",
+  "jobHeaders": {
+    "workflowInstanceKey": 6,
+    "bpmnProcessId": "order-process",
+    "workflowDefinitionVersion": 1,
+    "workflowKey": 1,
+    "activityId": "ship-parcel",
+    "activityInstanceKey": 41
+  },
+  "customHeaders": "{}",
+  "worker": "zbctl",
+  "retries": 3,
+  "deadline": 1539604831656,
+  "payload": "{\"orderId\":1234}"
 }
 ```
 
-As you can see in the event log the last event was a workflow instance event
-with the intent `COMPLETED`, which indicates that the instance
-we started in [step 8](#step-8-create-a-workflow-instance) was now completed.
```

Comment:
Maybe we can refer to the exporters? So if the user wants to get more insights what happened he can create an exporter  to see all events.